### DISCUSSION TITLE
I want to upload Node.js project to godaddy

### DISCUSSION THREAD

**Discussion by abdallaamrfawzy** [community/community#134725] [Programming Help] [answered]
### Body

I want to upload Node.js project to Godaddy
Should I choose VPS hosting or what
I always uploaded to render but this is the first time I have dealt with Godaddy
Does uploading to Godaddy require many steps and things that I need to understand

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by ClanLeader** [comment, marked as answer]
No, you don't need to use VPS, there are shared hosting that support Node.js too, for example Asphostportal. May I know have you asked it to Godaddy? Is there any reason why you use GD as your provider? My advice, you better stay away from this company.

**Reply by abdallaamrfawzy** [reply]
Why should I move away from this company? Can you explain?
I have already uploaded node.js to render.
But the client asked me to move to godady.

Can you explain why I don't deal with this company?
Is it specifically with node.js or not dealing with it in general?
So that I can explain that to the client

**Reply by ClanLeader** [reply]
It is simply because their support is pathetic, I believe that you can read about their reviews.

**Comment by deep-sengupta** [comment]
For uploading a Node.js project to GoDaddy, VPS hosting is a good choice because it provides the control and resources needed for a Node.js application. The process involves setting up your server, installing Node.js, and configuring your project, which can be more complex than using Render. However, there are many tutorials available to guide you through the steps.

**Comment by cucer** [comment]
I prefer Google Cloud - AppEngine instead of GoDaddy, it is very simple to deploy and configure basic settings. And of course price is reasonable
